pkg/assembler/helpers/vcs.go

```diff
@@ -0,0 +1,88 @@
+//
```

Comment:
Should this live in a common helpers directory? Looking at this, this function is more or less the same one we use in the github collector to parse a URI that comes in as well.

Reply:
yea, maybe a helper function to call out, and then leave the conversion to the graphql model in assembler helpers.

Reply:
i think this could be put into a good first issue and handled as another PR?

Reply:
https://github.com/guacsec/guac/issues/572